packages/admin/admin-theme/src/componentsTheme/MuiButton.ts

```diff
@@ -37,6 +38,15 @@ export const getMuiButton: GetMuiComponentTheme<"MuiButton"> = (component, { pal
                 marginLeft: 5,
             },
 
+            "&:hover": {
+                color: palette.primary.main,
+                backgroundColor: "transparent",
+            },
+
+            "&:focus": {
+                border: `2px solid ${palette.primary.dark}`,
```

Comment:
Adding a border causes layout-shift during focus, could you use `outline` instead? 

```suggestion
                outline: `2px solid ${palette.primary.dark}`,
```

Reply:
There's still some layout-shift, see the buttons below the `destructive` are moving when focusing different buttons. 

Also, the outline is now in addition to (outside of) the border. 
Maybe you need an `outlineOffset` to compensate for the border? 🤔 

Please make sure this is fixed for all button variants. 

https://github.com/user-attachments/assets/43c5a642-ff0f-45b5-b51c-9aae9485c39a